src/main/java/com/liefery/android/stop_badge/StopBadge.java

```diff
@@ -233,12 +233,15 @@ int shadowSizeX() {
     }
 
     int shadowSizeY() {
-        return (int) ( shadowRadius + Math.abs( shadowDx ) );
+        return (int) ( shadowRadius + Math.abs( shadowDy ) );
     }
 
     public Bitmap export( int size ) {
-        Bitmap bitmap = Bitmap.createBitmap( size + shadowSizeX() * 2, size
-            + shadowSizeY() * 2, Bitmap.Config.ARGB_8888 );
+        int newSize = Math.max( shadowSizeX(), shadowSizeY() ) * 2 + size;
```

Comment:
This makes sure the Bitmap is always square. This could be a issue in cases where the image view is not square and the scale type is not set to center. Our use cases are not affected, but others using the library differently my ay experience offset badges.  It's more like a precautionary "buxfix". I can remove it if you'd like to.